i18n/management/commands/publish_i18n.py

```diff
@@ -60,7 +65,7 @@ def publish_models(self):
                 success_count, total, name, datetime.timedelta(seconds=int(elapsed_time))
             ))
 
-        log("Publishing %s models in %s languages finished in %s (average of ~%s per language" % (
+        log("Publishing %s models in %s languages finished in %s (average of ~%s per language)" % (
```

Comment:
typo?

Reply:
Fixing the typo 
![Screenshot 2020-01-17 at 9 08 12 AM](https://user-images.githubusercontent.com/46464143/72631368-033c6500-3909-11ea-920e-f72481c19638.png)

Reply:
oooh, i totally misread this as the paren being outside the quote